app/assets/stylesheets/widgets/home_page/hero_panel.scss

```diff
@@ -258,17 +258,12 @@
   #hero {
```

Comment:
Could you please delete the empty rules in this file? (ex. `&.thin {}`, `#mastheadContent {}`) Unless there is a good reason for keeping them. That's our best practice.